crates/swc_ecma_parser/src/parser/expr.rs

```diff
@@ -1904,4 +1904,21 @@ impl<'a, I: Tokens> Parser<I> {
             }
         }
     }
+
+    fn is_start_of_left_hand_side_expr(&mut self) -> PResult<bool> {
+        Ok(is_one_of!(
+            self, "this", "super", "null", "true", "false", Num, BigInt, Str, '`', '(', '[', '{',
+            "function", "class", "new", '/', "/=", IdentRef
```

Comment:
Does `/=` start an expression?

Reply:
Also I'm not sure about `/`

Reply:
> Does `/=` start an expression?

I'm not sure, and I just refer to TypeScript's implementation.

> Also I'm not sure about `/`

It's regular expression, I think. It's also referred from TypeScript.

Reply:
For `/`, we use `Regex` instead.
https://docs.rs/swc_ecma_parser/latest/swc_ecma_parser/token/enum.Token.html#variant.Regex

Reply:
I think our lexer is much more smarter than the one of tsc.
To lex regex, we need the context of parser.
So there are

Reply:
So can you remove `/` and `/=` from the list?

Reply:
Sure, I will remove them later.